ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/CommandTransactionChecks.scala

```diff
@@ -44,6 +44,12 @@ import org.scalatest.Inside._
 import org.scalatest._
 import org.scalatest.concurrent.{AsyncTimeLimitedTests, ScalaFutures}
 import scalaz.syntax.tag._
+import com.digitalasset.daml.lf.language.{
+  LanguageVersion,
+  LanguageMajorVersion => LMajV,
+  LanguageMinorVersion => LMinV
```

Comment:
`LanguageVersion.Major` is a synonym for `LanguageMajorVersion`, and similar for Minor, which you may or may not wish to use as an alternative shortcut. (I'm not particularly attached to any style besides "don't be shy about renaming imports just because they're not in Java")